packages/typespec-ts/test/azureIntegration/generated/payload/pageable/src/index.d.ts

```diff
@@ -24,7 +22,147 @@ export declare interface PetOutput {
 }
 
 export declare interface Routes {
-    (path: "/payload/pageable/server-driven-pagination/link"): Link;
+    (path: "/payload/pageable/server-driven-pagination/link"): ServerDrivenPaginationLink;
+    (path: "/payload/pageable/server-driven-pagination/continuationtoken/request-query-response-body"): ServerDrivenPaginationContinuationTokenRequestQueryResponseBody;
```

Comment:
why we have this change?

Reply:
refer to : https://github.com/microsoft/typespec/pull/6140/files